db/seeds.rb

```diff
@@ -223,6 +223,11 @@
     ]
   end
 
+  ### USERS ###
+  User.create!(email: 'tom.black@agile6.com', password: 'Password123', password_confirmation: 'Password123').add_role(User::USER_ROLES[1].to_sym)
```

Comment:
if we're going to confirm email addresses, tom's needs to be correct: `tom@skylight.digital`